src/Numeric/Units/Dimensional/UnitNames/Internal.hs

```diff
@@ -141,12 +141,12 @@ data Prefix = Prefix
                 -- | The name of a metric prefix.
                 prefixName :: PrefixName,
                 -- | The scale factor denoted by a metric prefix.
-                scaleFactor :: Rational
+                scaleExponent :: Int
```

Comment:
I haven't thought of any other prefixes so you can leave as is. Regarding the binary prefixes I wouldn't want to pollute dimensional with them. I think they would be better handled by an ad hoc data type (outside of the scope of dimensional) rather than adding another dimension.

> On 25 aug. 2016, at 01:36, Douglas McClean <notifications@github.com> wrote:
> 
> In src/Numeric/Units/Dimensional/UnitNames/Internal.hs:
> 
> > @@ -141,12 +141,12 @@ data Prefix = Prefix
> >                  -- | The name of a metric prefix.
> >                  prefixName :: PrefixName,
> >                  -- | The scale factor denoted by a metric prefix.
> > -                scaleFactor :: Rational
> > +                scaleExponent :: Int
> Do you want me to redo this to work around the change to scaleExponent? Or leave it as is?
> 
> Even in the case of moving to binary prefixes we could conceivably keep this API and add scaleBase?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>